pom.xml

```diff
@@ -63,7 +63,10 @@
     </pluginRepositories>
     <properties>
         <jenkins.version>1.642.3</jenkins.version>
+        <jenkins-test-harness.version>2.19</jenkins-test-harness.version>
         <workflow-cps-plugin.version>2.28</workflow-cps-plugin.version>
+        <git-plugin.version>3.1.0</git-plugin.version>
```

Comment:
I really hate the use of a property here -- if you just move the normal and test deps adjacent it's trivial to bump both.  Use search+replace if you sometimes forget one.  This level of DRY is silly.